### PR TITLE
📚 Docs Refresh: Google Analytics Source

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Changes
- Reorganization
- Add reference to Google Analytics V4
- Add reference to Airbyte Cloud

### CODE REVIEW

File: docs/integrations/sources/googleanalytics.md [hunk 1]
```diff
@@ -1,44 +1,37 @@
 # Google Analytics
 
-## Overview
```
Comment (sherifnada) [inline docs/integrations/sources/googleanalytics.md:3]: we can remove this whole page because v4 is the only version supported going forward

**Review by sherifnada — COMMENTED** [review]
we should remove this page since v4 is the only supported connector now

File: docs/integrations/sources/googleanalytics.md [hunk 2]
```diff
@@ -1,44 +1,37 @@
...
-| Namespaces | No |
+| Custom Reports | Yes |
 
-### Performance considerations
+This Google Analytics source wraps the [Pipelinewise Singer Google Analytics Tap](https://github.com/transferwise/pipelinewise-tap-google-analytics).
 
-The Google Analytics connector should not run into Google Analytics API limitations under normal usage. Please [create an issue](https://github.com/airbytehq/airbyte/issues) if you see any rate limit issues that are not automatically retried successfully.
+## Supported Tables
+
+* website_overview
```
Comment (marcosmarxm) [inline docs/integrations/sources/googleanalytics.md:17]:
```suggestion
* `website_overview`
```

**Review by marcosmarxm — APPROVED** [review]
LGTM

File: docs/integrations/sources/googleanalytics.md (hunk 1 above)
Comment (avaidyanatha) [reply docs/integrations/sources/googleanalytics.md:3]: Cool, done - removed from SUMMARY.md first and will purge later.

File: docs/integrations/sources/googleanalytics.md (hunk 2 above)
Comment (avaidyanatha) [reply docs/integrations/sources/googleanalytics.md:17]: Removing the whole page, so this will be unnecessary!

**Pull request merged**